### ISSUE TITLE
Missing localisation for water purification plant achievements

### ISSUE THREAD

**Issue by combusterf** [GTNewHorizons/GT-New-Horizons-Modpack#18340]
### Your GTNH Discord Username

Combuster

### Your Pack Version

2.7.0

### Your Server

Delta

### Java Version

Java 17

### Type of Server

Thermos

### Your Expectation

Get localised achievements for waterline blocks

### The Reality

![image](https://github.com/user-attachments/assets/0eac433c-6178-4a65-aa07-216c6f952989)
![image](https://github.com/user-attachments/assets/dd1630d0-c40a-47d9-8d5a-c0fc79777220)

### Your Proposal

These have probably been forgotten. Please add them, the purification plant and at least T1-T2 have been confirmed affected so I expect the same to be true for the entire series. 

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by FourIsTheNumber** [comment]
This is true for most assembling line recipes. Since these are no longer necessary, automatically generated achievements for basic tasks should just be removed imo.
